src/cruise_literature/cruise_literature/settings.py

```diff
@@ -172,6 +172,6 @@
 # if True then also search inside CORE publications aggregator
 SEARCH_WITH_CORE = True
 
-TEXT_TO_TEXT_API = True
+ML_API = False
```

Comment:
did you maybe add info to readme how to enable `ML_API` in settings?

Reply:
We have updated README